pyproject.toml

```diff
@@ -75,7 +74,6 @@ include-package-data = true
 version = {attr = "trio._version.__version__"}
 
 [tool.black]
-target-version = ['py38']
```

Comment:
why not specify target-version anymore?

Reply:
It gets inferred

Reply:
See https://github.com/python-trio/trio/pull/3106#discussion_r1803919180

Reply:
neat!